src/Orchard.Web/Core/Common/Handlers/CommonPartHandler.cs

```diff
@@ -97,6 +97,7 @@ public class CommonPartHandler : ContentHandler {
             part.ModifiedUtc = utcNow;
             part.VersionCreatedUtc = utcNow;
             part.VersionModifiedUtc = utcNow;
+            part.VersionModifiedBy = GetUserName();

```

Comment:
Look for unit tests on `VersionModifiedUtc` and please add the same ones on `VersionModifiedBy`.